Makefile.build

```diff
@@ -765,6 +765,21 @@ $(VDFILE).d: $(D_DEPEND_BEFORE_SRC) $(VFILES) $(D_DEPEND_AFTER_SRC) $(COQDEPBOOT
 	$(SHOW)'COQDEP    VFILES'
 	$(HIDE)$(COQDEPBOOT) -boot $(DYNDEP) $(VFILES) $(TOTARGET)
 
+###########################################################################
+# Bisect Coverage
+###########################################################################
+.PHONY: bisect_report
+
+bisect_report: coveralls.json
+
+BISECT_FILES=bisect*.out
+# use ocveralls --send ?
+coveralls.json: OCAMLC := $(OCAMLFIND) ocamlc $(CAMLFLAGS) -package bisect_ppx
+coveralls.json: OCAMLOPT := $(OCAMLFIND) opt $(CAMLFLAGS) -package bisect_ppx
+coveralls.json: test-suite
+	ocveralls $(BISECT_FILES) > coveralls.json
+	bisect-ppx-report -html coverage/ $(BISECT_FILES)
```

Comment:
This line makes `coverage/index.html` (and others) and so is in the wrong target.

Reply:
`coverage/index.html` is just an extra.

Reply:
>coverage/index.html is just an extra.

So?